### ISSUE TITLE
Cannot toggle lightpad via keyboard shortcut

### ISSUE THREAD

**Issue by fossfreedom** [libredeb/lightpad#9]
Suggested enhancement.

as per the README - CTRL+SPACE will launch lightpad - but you cannot CTRL+SPACE again to close.

This means you have to move to the escape key to close.  This is an unusual behaviour for many desktop environments where you can toggle launchers on and off.

**Comment by libredeb** [comment]
@fossfreedom yes... I understand the point. We can only remap the ESC key to hide LightPad and leave the combination keys to open and close to the user's choice. What do you think?

**Comment by marizmelo** [comment]
+1 
I am using the SUPER key to activate lightpad, it would be helpful that whatever the option to open it (SUPER, Ctrl+space, etc) I could press it again to dismiss the application.

**Comment by libredeb** [comment]
@marizmelo @fossfreedom Done! 😊 The issue was resolved in this [pull-request #42](https://github.com/libredeb/lightpad/pull/42). Regards!
